### PR TITLE
WIP Add benchmarks to github actions

### PR DESCRIPTION
**Pull request by ry**

### CODE REVIEW

**Comment by ry** [comment]
This almost working - but I'm having trouble getting github pages to upload correctly.

File: .github/workflows/build.yml [hunk 1]
```diff
@@ -73,7 +79,29 @@ jobs:
...
+        if: matrix.kind == 'bench' && (github.ref == 'refs/heads/master' || github.ref == 'refs/pull/3008/merge')
+        env:
+          GH_PAT: ${{ secrets.GH_PAT }}
+        run: |
+          # Note gh-pages branch is cloned into //gh-pages/ by
+          # tools/benchmark.py, hence the following copy is ok:
+          cp -r website/* gh-pages/
+          cd gh-pages
+          # git remote add origin-ssh git@github.com:denoland/deno.git
+          git remote add origin2 https://$GH_PAT@github.com/denoland/deno.git
```
Comment (hayd) [inline .github/workflows/build.yml]: username:password@
username:token@

Comment (ry) [reply .github/workflows/build.yml]: good idea - but didn't work
https://github.com/denoland/deno/pull/3008/checks?check_run_id=237718301
```
remote: Invalid username or password.
fatal: Authentication failed for 'https://github.com/denoland/deno.git/'
```

Comment (hayd) [reply .github/workflows/build.yml]: $GH_PAT is an access token for this user?

Comment (ry) [reply .github/workflows/build.yml]: Yeah... I actually tried this locally and it works...

**Comment by ry** [comment]
closed in favor of https://github.com/denoland/deno/pull/3049

The problem was that the access token isn't available to forks - and this PR was using a @ry fork.  #3049 is an origin branch - it gets the access token properly.

**Pull request closed without merging**